webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require ('webpack')
+
+module.exports = {
+    entry: './source/likely.js',
+    output: {
+        filename: 'likely.js',
+        library: 'likely',
+        libraryTarget: 'umd',
+    },
+    plugins: [
+        new webpack.optimize.OccurrenceOrderPlugin(),
+        new webpack.optimize.DedupePlugin(),
+        new webpack.optimize.UglifyJsPlugin(),
```

Comment:
Found it, it’s called `screw_ie8` and is set like here: https://github.com/SimenB/redux/blob/0b9f9ec2adcf642a1be761763ba3a99b428a3e35/webpack.config.js#L31